src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -527,13 +527,25 @@ def runK8(image, branchName, config, axis) {
     def runAsUser = image.runAsUser ?: getConfigVal(config, ['kubernetes', 'runAsUser'], "0")
     def runAsGroup = image.runAsGroup ?: getConfigVal(config, ['kubernetes', 'runAsGroup'], "0")
     def privileged = image.privileged ?: getConfigVal(config, ['kubernetes', 'privileged'], false)
+    def limits = image.yaml ?: getConfigVal(config, ['kubernetes', 'limits'], "")
+    def requests = image.yaml ?: getConfigVal(config, ['kubernetes', 'requests'], "")
+    def yaml = """
```

Comment:
also in line 873
https://github.com/Mellanox/ci-demo/pull/15/files#diff-c681df6da39890ea69610534647a729d5b1e0738e539c594571113c2e4e2b012R873

Reply:
hmm.. it is for docker build flow, maybe not needed. WDYT

Reply:
This function is only used to build the image and not to run, so we don't need to provide `yaml` options for it.
I've also tested it to be sure (http://hpc-master.lab.mtl.com:8080/job/mlnx-snap-andriih/79/console)